### DISCUSSION TITLE
OCTOPOD for android

### DISCUSSION THREAD

**Discussion by brolly916** [gdombiak/OctoPod#569] [General]
Hello can someone guide me to from where i can download the Octopod apk for android please ?

**Comment by gdombiak** [comment]
Hi @brolly916,

OctoPod is only for iOS. Check out Printoid as a great Android alternative. Printoid also offers a similar OctoPrint plugin like OctoPod for receiving notifications on your phone for important events.

Good luck,
Gaston
